shell/browser/ui/views/frameless_view.cc

```diff
@@ -112,6 +112,16 @@ void FramelessView::UpdateWindowTitle() {}
 
 void FramelessView::SizeConstraintsChanged() {}
 
+views::View* FramelessView::TargetForRect(views::View* root,
+                                          const gfx::Rect& rect) {
+  CHECK_EQ(root, this);
```

Comment:
```suggestion
  DCHECK_EQ(root, this);
```
Can this be turned into a debug check or is there any chance that user code can trigger an assertion failure here (in that case, shouldn't we turn this into an exception instead of a hard crash)?

Reply:
it's probably better we act a much like [upstream](https://source.chromium.org/chromium/chromium/src/+/main:ui/views/window/non_client_view.cc;l=318?q=%22::TargetForRect(%22&ss=chromium) here as possible.

Reply:
Ah, didn't realize that this was just mirroring what the upstream implementation was doing. This looks okay to me then. :+1: